src/constants.ts

```diff
@@ -14,8 +14,9 @@ export enum MetaMetricsEventName {
   StxStatusUpdated = 'STX Status Updated',
   StxConfirmed = 'STX Confirmed',
   StxConfirmationFailed = 'STX Confirmation Failed',
+  ReceiveRequest = 'Receive Request',
 }
-
```

Comment:
Nit: can we return back the empty line here

Reply:
Done